packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.tsx

```diff
@@ -583,7 +584,7 @@ const MarkdownSynapse: MarkdownSynapseComponent = class MarkdownSynapse extends
     )
   }
 
-  public async componentDidMount() {
+  public componentDidMount(): void {
```

Comment:
New lint rule bans inherited methods from returning a promise when the interface does not.